tests/kernel/mem_protect/syscalls/testcase.yaml

```diff
@@ -5,4 +5,4 @@ tests:
     # FIXME: This test may fail for qemu_arc_em on certain host systems.
     #        See foss-for-synopsys-dwc-arc-processors/qemu#66.
     platform_exclude: qemu_arc_em
-    timeout: 180
+    timeout: 480
```

Comment:
I know this may be too long, but FVP with 4 cores simulation is very slow ( 20 ~ 40 times slower depending on the machine).